spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'User show method ', type: :feature do
+  # rubocop:disable Layout/LineLength
+  before(:example) do
+    @user = User.create(name: 'Moha',
+                        photo: 'https://images.unsplash.com/photo-1594897030264-ab7d87efc473?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2940&q=80',
+                        bio: 'Teacher from Yemen.', posts_count: 8)
+
+    @posts = [Post.create(author: @user, title: 'Hi', content: 'This is my first post', commentsCounter: 0, likes_count: 0),
+              Post.create(author: @user, title: 'Hey', content: 'This is my second post', commentsCounter: 0, likes_count: 0),
+              Post.create(author: @user, title: 'What\'s up', content: 'This is my third post', commentsCounter: 0, likes_count: 0)]
+    # rubocop:enable Layout/LineLength
+    visit user_path(@user)
+  end
```

Comment:
- [ ] It would be nice if you don't disable your linters and try to fix them manually because linters prevent you from making those errors which can break your app completely 😨 Kindly remove these linters disabling lines and fix the errors manually 👍